src/associated_token_account.zig

```diff
@@ -128,3 +129,42 @@ pub fn createIdempotentAccount(account: sol.Account.Info, params: struct {
         params.associated_token_program,
     }, params.seeds);
 }
+
+pub fn recoverNestedAccount(allocator: std.mem.Allocator, account: sol.Account.Info, params: struct {
+    nested_mint: sol.Account.Info,
+    destination_associated_account: sol.Account.Info,
+    owner_associated_account: sol.Account.Info,
+    owner_mint: sol.Account.Info,
+    owner: sol.Account.Info,
+    token_program: sol.Account.Info,
+    associated_token_program: sol.Account.Info,
+    seeds: []const []const []const u8 = &.{},
+}) !void {
```

Comment:
nit: let's put all of the accounts under the `params` struct
```suggestion
pub fn recoverNestedAccount(allocator: std.mem.Allocator, params: struct {
    account: sol.Account.Info
    nested_mint: sol.Account.Info,
    destination_associated_account: sol.Account.Info,
    owner_associated_account: sol.Account.Info,
    owner_mint: sol.Account.Info,
    owner: sol.Account.Info,
    token_program: sol.Account.Info,
    associated_token_program: sol.Account.Info,
    seeds: []const []const []const u8 = &.{},
}) !void {
```

Reply:
done ✅